src/app/pages/system/update/components/train-info-card/train-info-card.component.html

```diff
@@ -0,0 +1,52 @@
+<mat-card
+  *ngIf="(updateService.updatesAvailable$ | async) && (trainService.preReleaseTrain$ | async) && productType === ProductType.ScaleEnterprise"
```

Comment:
Let's combine observables in the controller. Also, there is an `isEnterpise$` observable available in `systemGeneralService.`